### PR TITLE
Draft release notes for beta-20161110

### PR DESCRIPTION
**Pull request by bdarnell**
As of commit cockroachdb/cockroach@954063f7

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/835)

### CODE REVIEW

File: beta-20161110.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+## Nov 10, 2016
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161110.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161110.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### Upgrade Notes
```
Comment (jseldess) [inline beta-20161110.md]: I think titling this section "Backwards-Incompatible Change", as we've done in the past, would make this stand out more. But then we might want to write a sentence or two about the nature of this change. What do you think, @bdarnell?

**Comment by mberhault** [comment]
CC: @a-robinson 
it looks like this sha failed the release build on teamcity, we may want to force it. I'm having trouble with DNS for all `cockroachdb.com` things right now, so I can't do it :(

**Comment by a-robinson** [comment]
A retry has been forced.

**Comment by bdarnell** [comment]
The release notes always fail teamcity when written because the links point to tarballs that don't exist yet.

**Comment by a-robinson** [comment]
@mberhault was referring to [the build of the actual release](https://teamcity.cockroachdb.com/viewLog.html?buildId=43767&tab=buildResultsDiv&buildTypeId=Cockroach_MergeToMaster) though, which meant that a binary for the SHA hadn't been pushed out to S3. In fact it still hadn't been pushed as of this morning, so I have [a new attempt](https://teamcity.cockroachdb.com/viewLog.html?buildId=44984&) running to hopefully resolve that.

**Comment by a-robinson** [comment]
Ok, now I'm confused about how to get a binary pushed out for this particular sha so that we can get it tested. The restarted teamcity run succeeded, but there's still no binary with that SHA. I'm sure I'm just missing some basic piece of the process, but can anyone give a pointer to help save me some time?

**Comment by mberhault** [comment]
the attempt you point to is for sha f04c72ec6dc32b0379e0df7f3be0b58d37e52c67, the earlier one (that failed) is for 954063f71de001d4861d62592779408fa120dbb3

**Comment by a-robinson** [comment]
Ah, apparently clicking the "Run" button on the failed attempt is not the way to force it because by default it'll pick up a new sha :/ Thanks Marc. Retrying.

File: beta-20161110.md (hunk 1 above)
Comment (jseldess) [inline beta-20161110.md]: I see your point. I'll stick with Upgrade Notes then.

**Pull request merged**